scripts/zones/Empyreal_Paradox/npcs/Transcendental.lua

```diff
@@ -21,6 +21,8 @@ function onTrigger(player,npc)
     --player:setCharVar("PromathiaStatus",3);
     if (player:getCurrentMission(COP) == tpz.mission.id.cop.DAWN and player:getCharVar("PromathiaStatus")==1) then
         player:startEvent(2);
+	elseif (player:getQuestStatus(JEUNO,tpz.quest.id.jeuno.APOCALYPSE_NIGH) == QUEST_ACCEPTED and player:getCharVar('ApocalypseNigh') == 3) then	
```

Comment:
tabs (not gonna keep mentioning them, but tabs continue to be an issue in later files)